Fix The HTML `<Meter>` element example #34413

Ashish-CodeJourney opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->
### Description
<!-- ✍️ Summarize your changes in one or two sentences -->
Fix the HTML example for the `<meter>` element to correctly display the text after the meter.

### Motivation
<!-- ❓ Why are you making these changes and how do they help readers? -->
The current example didn't render as intended, potentially confusing readers about the proper usage of the `<meter>` element. This change provides a more accurate and helpful example.

### Additional details
<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->
The issue was observed in Google Chrome. The current HTML:

```html
<p>
  Battery level: <meter min="0" max="100" value="75">75%</meter>
</p>
```

should be changed to:

```html
<p>
  Battery level: <meter min="0" max="100" value="75"></meter> 75%
</p>
```

This change is consistent with clearer rendering and better aligns with usage expectations.

### Related issues and pull requests
<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->
<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->
Fixes #34413

Ashish-CodeJourney committed:
Fix The HTML `<Meter>` element example

Josh-Cena commented:
This is not what https://github.com/mdn/content/issues/34413#issuecomment-2190316590 said should be done, but I'll let @estelle decide.

estelle commented on files/en-us/web/html/element/meter/index.md:
````diff
@@ -42,7 +42,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 ```html
 <p>
-  Heat the oven to <meter min="200" max="500" value="350">350 degrees</meter>.
+  Heat the oven to <meter min="200" max="500" value="350"></meter> 350 degrees.
````
The original was intended. The text between the opening and closing tag was put in there for browsers that did not support the `<meter>` element . Now that `<meter>` is supported everywhere, the text can be removed.

The text should not be put after the element as the meter value may change, and the text has no way of being updated to reflect any change.

estelle replied:
As @Josh-Cena wrote in the issue, a good solution would be to either replace the example with a useful one or to add within the example description that the text is meant as a fallback text for (a) screen readers (b) browsers that don't support meter bars (c) any other cases when the bar doesn't show.

Ashish-CodeJourney replied:
@estelle @Josh-Cena if you can just review mew example, i guess this makes much more sense

Ashish-CodeJourney committed:
Merge branch 'main' of https://github.com/Ashish-CodeJourney/content into feature/34413

Ashish-CodeJourney committed:
Update `<meter>` element examples for realism and clarity

Refined examples for the `<meter>` element to offer more practical use cases:
- Updated the simple example to showcase a battery level meter.
- Revised the high and low range example to demonstrate a student's exam score meter.

Included corresponding screenshots for the new examples to enhance clarity and understanding.

Ashish-CodeJourney committed:
Update files/en-us/web/html/element/meter/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

estelle commented on files/en-us/web/html/element/meter/index.md:
```diff
@@ -52,17 +52,18 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
+![A screenshot of the meter element in Google Chrome](Screenshot_from_2024-06-27_10-40-55.png)
 
-### High and Low range example
+### High and Low Range Example
```
```suggestion
### High and low range example
```
Links and headers across the site are sentence case (here and below)

estelle commented on files/en-us/web/html/element/meter/index.md:
```diff
@@ -52,17 +52,18 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
 On Google Chrome, the resulting meter looks like this:
 
-![A screenshot of the meter element in Google Chrome](screen_shot_2020-10-12_at_10.10.53_pm.png)
```
As meter is well supported in all browsers now, we can remove the screenshots altogether and use live examples instead.

estelle reviewed (commented):
Titles and links on MDN are sentence case, per the MDN writing style guidelines.

Also, meter is well supported. Instead of updating the images, which was done for when meter did not have full support, let's use live examples instead.

Ashish-CodeJourney committed:
Update files/en-us/web/html/element/meter/index.md

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

estelle committed:
removing images

the meter element is supported everywhere, so screenshots are no longer needed

Ashish-CodeJourney commented:
@estelle  thanks for cooperation
i am occupied with some urgent work load thats why i was nopt  to fix that

estelle committed:
fix lint issue

estelle committed:
delete images

estelle commented:
> @estelle thanks for cooperation i am occupied with some urgent work load thats why i was nopt to fix that

I hope everything is ok.
if you merge https://github.com/Ashish-CodeJourney/content/pull/3, it will update this PR and make this one mergeable

Ashish-CodeJourney committed:
Merge branch 'main' into feature/34413

Ashish-CodeJourney committed:
Merge pull request #3 from mdn/meter-image-removal

delete images

Ashish-CodeJourney commented:
> > @estelle thanks for cooperation i am occupied with some urgent work load thats why i was nopt to fix that
> 
> I hope everything is ok. if you merge [Ashish-CodeJourney#3](https://github.com/Ashish-CodeJourney/content/pull/3), it will update this PR and make this one mergeable

Done, Thanks :)

estelle reviewed (approved):
Thanks! 🎉
